LayoutTests/fast/canvas/canvas-filter-text-drawing.html

```diff
@@ -0,0 +1,20 @@
+]<meta name="fuzzy" content="maxDifference=0-255; totalPixels=0-5627" />
```

Comment:
Finger fumble here.

That's huge fuzziness. Up to 5627 pixels can be completely different?

Reply:
Fixed by drawing an SVG which has the filtered text instead of drawing the text on a detached canvas and then applying the filter to it.